.github/workflows/ci.yaml

```diff
@@ -107,9 +71,48 @@ jobs:
       - uses: satackey/action-docker-layer-caching@v0.0.11
         # Ignore the failure of a step and avoid terminating the job.
         continue-on-error: true
+      # Rerun the test to be able to publish all coverage reports in the same flow
+      - name: Run packages test
+        run: npm run test -- --force
```

Comment:
cant we cache the previous results somehow and still make this work?

Reply:
This fixes a problem we had in the previous PR: not all the test are rerun (depending on the changes done in the PR), that means potentially not all the coverage reports are published to CodeCov.
Here, because the PR does not change any source code of the packages, no package coverage report have been published and the overall score was just decreasing compared to main, blocking the merge (bad score from CodeCov).